docs/kcl_mod-zh.md

```diff
@@ -0,0 +1,96 @@
+# kcl.mod: KCL 包清单文件
+
+## KCL 包清单
+
+每个模块的 `kcl.mod` 文件都被称为其清单。它采用 TOML 格式编写，包含编译模块所需的元数据。
+
+目前 `kcl.mod` 中支持如下内容：
+
+- 包元数据：
```

Comment:
You can organize the entire document by field type, description, default value, required or optional, and use case.